src/prototypes/example/summary.njk

```diff
@@ -24,8 +24,8 @@ layout: example/views/_main
     <div class="ons-summary">
         <div class="ons-summary__group">
             <h2 class="ons-summary__group-title">Turnover</h2>
-            <table id="turnover" class="ons-summary__items js-summary">
-            </table>
```

Comment:
This shouldn't be changed to `dl` due to the current configuration. The change you made is causing it to break.